tests/benchmark/numpy_dtypes.py

```diff
@@ -61,14 +61,14 @@
     }),
     'np.string': pd.DataFrame({
         'np.string': pd.Series([
-            np.string_('string1'),
-            np.string_('string2'),
-            np.string_('string3'),
+            np.bytes_('string1'),
+            np.bytes_('string2'),
+            np.bytes_('string3'),
```

Comment:
Can we use `str_` instead ? And duplicate this to `np.bytes` or something and add the same strings but with `bytes` instead.